src/backends/torch/torchmodule.h

```diff
@@ -50,10 +50,20 @@ namespace dd
 
     void freeze_traced(bool freeze);
 
-    /** Add linear model at the end of module. Automatically detects size of
+    void setup_linear_layer(int nclasses,
+                            std::vector<c10::IValue> input_example);
+
+    /** Add linear layer at the end of module. Automatically detects size of
      * the last layer thanks to the provided example output.*/
-    void setup_classification(int nclasses,
-                              std::vector<c10::IValue> input_example);
+    /*void setup_classification(int nclasses,
+      std::vector<c10::IValue> input_example);*/
```

Comment:
Is something planned with these methods, or can they be removed ?

Reply:
good catch, done.